app/assets/javascripts/components/forms/calendar.jsx

```diff
@@ -1,25 +1,34 @@
-import React, { Component } from 'react'
+import React, { PureComponent } from 'react'
+import { Field } from 'redux-form';
 import { DateField } from 'react-date-picker'
+import moment from 'moment';
 
-export default class Calendar extends Component {
-  constructor(props) {
-    super(props)
-    this.handleChange = this.handleChange.bind(this);
-    this.state = {date: this.props.date};
-  }
+class Calendar extends PureComponent {
+  render() {
+    const {
+      input,
+      valid,
+      meta: { touched, error} ,
+      ...props,
+    } = this.props;
 
-  handleChange(e) {
-    const key = 'expirationDate'
-    this.setState({date: e});
-  }
+    const inputValue = moment(input.value) || moment();
```

Comment:
Can you make this 1 week away? Doesn't make sense to make it expire the same that you make the custom response.